src/components/input/TextComponent.tsx

```diff
@@ -160,18 +161,15 @@ const TextComponent: React.FC<TextComponentPropsWithType> = ({
             ),
           })
           : null}
-        {type == Types.TEXTFIELD ? (
+        {type == Types.TEXTFIELD && masked && value && value.length &&  (
```

Comment:
FYI : you can **value?.length** directly

Reply:
sure, done